packages/popover/src/Popover.stories.tsx

```diff
@@ -177,21 +189,42 @@ export const LiveExample: StoryFn<PopoverStoryProps> = ({
   buttonText,
   ...args
 }: PopoverStoryProps) => {
+  const {
+    portalClassName,
+    portalContainer,
+    portalRef,
+    scrollContainer,
+    usePortal,
+    ...rest
+  } = args;
+  const buttonRef = useRef<HTMLButtonElement | null>(null);
   const [active, setActive] = useState<boolean>(false);
 
   const position = referenceElPositions[refButtonPosition];
 
+  const handleToggle = (e: ToggleEvent) => {
+    const newActive = e.newState === 'open';
+    setActive(newActive);
+  };
+
   return (
-    <div className={regularStyles}>
+    <div className={containerStyles}>
       <Button
         className={cx(buttonStyles, position)}
         onClick={() => setActive(active => !active)}
+        ref={buttonRef}
       >
         {buttonText}
-        <Popover {...args} active={active}>
-          <div className={popoverStyle}>Popover content</div>
-        </Popover>
       </Button>
+      <Popover
+        {...rest}
+        active={active}
+        renderMode={RenderMode.TopLayer}
```

Comment:
technically we should probably open up the `renderMode` as a control in the LiveExample